test/cypress/integration/request-to-go-live/organisation_name_spec.js

```diff
@@ -36,6 +38,7 @@ describe('Request to go live: organisation name', () => {
       cy.get('#request-to-go-live-organisation-name-form').should('exist')
 
       cy.get('input#request-to-go-live-organisation-name-input').should('exist')
+      cy.get('input#request-to-go-live-organisation-name-input').invoke('val', 'Government Digital Service')
```

Comment:
The behaviour of this test has now changed and isn't testing what it was before, i.e. that if the service already has an organisation name set, it is pre-filled on the page.
You'll have to add another service to the self_service_user configuration to allow for testing this behaviour 😿

Reply:
Thanks, i'll update this 👍